### PR TITLE
ci: update fuzzer.sh script

### PR DESCRIPTION
**Pull request by Magorsky**
- Update scritp to create logs only on application fail

@sergeyfilyanin

### CODE REVIEW

**Comment by breathx** [comment]
@sergeyfilyanin kindly reminder for review

File: docker/runtime-fuzzer/scripts/fuzzer.sh [hunk 1]
```diff
@@ -2,9 +2,32 @@
 set -e
 
 VOLUME_DIR='/home/ubuntu/fuzzing-seeds-dir/'
+#ARCHIVE_NAME="node-fuzzer_logs_$(date +%Y-%m-%d_%H-%M-%S).tar.gz"
```
Comment (shamilsan) [inline docker/runtime-fuzzer/scripts/fuzzer.sh:5]: Do we need this commented line?
```suggestion
```

**Comment by shamilsan** [comment]
The PR seems to be staled. Reopen if needed.

File: docker/runtime-fuzzer/scripts/fuzzer.sh (hunk 1 above)
Comment (Magorsky) [reply docker/runtime-fuzzer/scripts/fuzzer.sh:5]: no, but left as an example

**Pull request closed without merging**